sdk/loadtestservice/Azure.ResourceManager.LoadTesting/README.md

```diff
@@ -44,6 +54,121 @@ Documentation is available to help you learn how to use this package:
 
 ## Examples
 
+### Create a new Azure Load Testing resource
```

Comment:
I want to share the process about how we add samples (use samples in Cdn as example).

- Add a [cs](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cdn/Azure.ResourceManager.Cdn/tests/Samples/Sample1_ManagingCdnOriginGroups.cs) file under tests/Samples to store the sample code
- Add the [tests](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cdn/Azure.ResourceManager.Cdn/tests/Samples/Sample1_ManagingCdnOriginGroups.cs#L20-L58) to show how to CRUD the sample resource and mark the tests as ignored.
- Use [#region](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cdn/Azure.ResourceManager.Cdn/tests/Samples/Sample1_ManagingCdnOriginGroups.cs#L25) and [#endregion](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cdn/Azure.ResourceManager.Cdn/tests/Samples/Sample1_ManagingCdnOriginGroups.cs#L57) to organize the sample code block and put the same [region name](https://github.com/Azure/azure-sdk-for-net/blame/main/sdk/cdn/Azure.ResourceManager.Cdn/samples/Sample1_ManagingCdnOriginGroups.md#L38) in the README file
- run `eng\scripts\Update-Snippets.ps1 loadtestservice` to update the samples in README

Could we please also follow the same process here? It ensures that our sample code could compile successfully and is up to date.